sys/compat/freebsd64/freebsd64_util.h

```diff
@@ -107,8 +107,8 @@ int	freebsd64_copyout_strings(struct image_params *imgp,
 	    uintcap_t *);
 int	freebsd64_copyiniov(struct iovec64 * __capability iovp, u_int iovcnt,
 	    struct iovec **iov, int error);
-int	freebsd64_copyinuio(struct iovec64 * __capability iovp, u_int iovcnt,
-	    struct uio **uiop);
+int	freebsd64_copyinuio(const /* struct iovec64 */void * __capability iovp,
```

Comment:
```suggestion
int	freebsd64_copyinuio(const /* struct iovec64 */ void * __capability iovp,
```
is the style used in a couple of other places.